src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,31 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        LocalDate startDate = LocalDate.parse(dateFrom, formatter);
+        LocalDate endDate = LocalDate.parse(dateTo, formatter);
+        StringBuilder builder = new StringBuilder("Report for period " + dateFrom + " - " + dateTo);
+
+        for (String name : names) {
+            int salary = 0;
+            for (String d : data) {
+                String[] split = d.split(" ");
```

Comment:
please avoid using one- or two-letters names, you will never remember what it means after couple of days